tests/model_hub_tests/pytorch/requirements.txt

```diff
@@ -26,3 +26,34 @@ transformers
 wheel
 PyYAML
 kornia
+
+openvino-tokenizers
+# use latest released version once it's available
+git+https://github.com/huggingface/optimum-intel.git@main
+pytest
+pytest-html
+# set 'export HF_HUB_ENABLE_HF_TRANSFER=1' to benefits from hf_transfer
+hf_transfer
+
+# requirements for specific models
+# - hf-tiny-model-private/tiny-random-RoFormerForCausalLM
+rjieba
+# - baichuan-inc/Baichuan2-7B-Chat
+bitsandbytes
+# - nomic-ai/gpt4all-falcon
+# - Qwen/Qwen-7B
+# - Qwen/Qwen-7B-Chat
+# - mosaicml/mpt-7b
+# - internlm/internlm2-7b
+einops
+# - Qwen/Qwen-7B
+# - Qwen/Qwen-7B-Chat
+transformers_stream_generator
+# - openbmb/MiniCPM-V-2
+torchvision
+# - openbmb/MiniCPM-V-2
+timm
+# - Qwen/Qwen-7B
+# - Qwen/Qwen-7B-Chat
+# - Salesforce/xgen-7b-8k-base
+tiktoken
```

Comment:
not sure that all are needed here.
having more deps creates a high risk of incompatibilities between deps

Reply:
Yeah, but we need them for the particular models

Reply:
I didn't find Qwen/Qwen* in your model list

Reply:
Yes, but we're planning to add more models to the pipeline later.

Reply:
Let us add on demand. Now it is not needed so pls remove. No value from them at this current moment

Reply:
Alright, agree. Removed.